metadata.yaml

```diff
@@ -17,6 +17,14 @@ tags:
   - storage
   - misc
 
+extra-bindings:
+  # Cluster public services bind to this space.
```

Comment:
Maybe could extend this in the comment a little, e.g.

```
# Cluster public services bind to this space for consumption by Ceph clients
```
I've always found the Ceph nomenclature a bit dangerous here, as `public` could imply exposing this to the wider internet.

Reply:
ack